client/components/seo/twitter-preview/index.jsx

```diff
@@ -30,7 +30,7 @@ export class TwitterPreview extends PureComponent {
 					<div className="twitter-preview__body">
 						<div className="twitter-preview__title">{ title }</div>
 						<div className="twitter-preview__description">{ description }</div>
-						<div className="twitter-preview__url">{ baseDomain( url ) }</div>
+						<div className="twitter-preview__url">{ baseDomain( url || '' ) }</div>
```

Comment:
I added because the component errors without a string being passed to `baseDomain`. All the other components handle this gracefully so this provides some consistency.